portfolio/src/main/webapp/style.css

```diff
@@ -568,7 +568,7 @@ button:hover {
 }
 
 /*Form*/
-#recommendations {
+form {
```

Comment:
This and below will set styles to all forms globally in your web page. Probably find if you only have 1 form now. But please consider if that's intended.

Reply:
Yes, it's intended for now since I added a new form that posts to datastore and kept my old form that posts to google sheets, and I wanted the new one to also adopt the same look. I will merge these two forms together later, and change this to a more specific id selector.

Reply:
Nevermind, I just changed the selector to the IDs of both forms.